woodwork/indexers.py

```diff
@@ -42,16 +42,14 @@ def _process_selection(selection, original_data):
         elif _is_dataframe(original_data):
             # Selecting a single column from a DataFrame
             schema = original_data.ww.schema.columns[selection.name]
-            schema.semantic_tags = schema.semantic_tags - {'index'} - {'time_index'}
+            schema.semantic_tags = (schema.semantic_tags - {'index'} - {'time_index'})
+            if schema.use_standard_tags:
+                schema.semantic_tags |= schema.logical_type.standard_tags
```

Comment:
Don't think I'm following this. Wouldn't the schema for the column already contain the standard tags?

Reply:
Not if it was previously an index column!

We make a point on line 45 to remove index tags, so if we don't manually add the standard tags back in, we lose them. Previously, we were going through the Column Accessor init, so the standard tags were getting added back in through that and we lose that when we init with a ColumnSchema

Reply:
Got it! Thanks for clarifying!